### ISSUE TITLE
how to output the result in coco json file and retrain a new model

### ISSUE THREAD

**Issue by HiAleeYang** [stefanopini/simple-HRNet#95]
Thank you for your work! It really make mine work easiler!
But now I have a problem. I have some RGB image and its corresponding silhouette image. I want to use HRnet to predict the RGB keypoint which is regarded as my Ground True Lable for silhouette image.And fine tuning the pretrained HRnet model on the silhouette while using the Predicted label(From RGB)
Is there a tutorial of that in Simple-HRnet?
Or can you give me some suggestions?(how to output the result in a resonable format and load the label to the new model)
THANK YOU VERY MUCH!

**Comment by HiAleeYang** [comment]
Is there a possible way just to use the keypoint as the G.T label?

**Comment by stefanopini** [comment]
Hi, I'm happy you found the project useful!

There was some discussion in the past on how to train on custom datasets (see issue #56 ).

I'm not aware of any easy way to convert the network outputs in the COCO dataset format, but you can save them in any format and create a custom training dataset that loads them, keeping in mind that

> Basically, if you want to use the train_coco.py script, you need a __getitem__ method that returns a tuple containing
image, target, target_weight, joints_data and a evaluate_accuracy method that returns accs, avg_acc, cnt, joints_preds, joints_target.

It should be relatively simple and much easier than trying to format your dataset in the COCO format imho.

Let me know if that helps 🙂
